components/notification/demo/duration.md

```diff
@@ -1,23 +1,32 @@
 ---
 order: 1
-title: 自动关闭的延时
+title: 
+  zh-CN: 自动关闭的延时
+  en-US: A duration to close notification box 
 ---
 
+## zh-CN
+
 自定义通知框自动关闭的延时，默认`4.5s`，取消自动关闭只要将该值设为 `0` 即可。
 
+## en-US
+
+To customize a duration to close notification, default value is `4.5s`, if you set the value to `0`,
+the corresponding notification box will be never closed automatically.
+
 ````jsx
 import { Button, notification } from 'antd';
 
 const openNotification = function () {
   const args = {
-    message: '这是标题',
-    description: '我不会自动关闭，我不会自动关闭，我不会自动关闭，我不会自动关闭，我不会自动关闭，我不会自动关闭，我不会自动关闭',
+    message: 'This is the title',
+    description: 'I will be never closed automatically.I will be never closed automatically.I will be never closed automatically.',
```

Comment:
description: 'I will never close automatically. You have to click on (x) button to close me.',